pkg/kubelet/kubelet.go

```diff
@@ -738,15 +738,15 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 			RelistPeriod:    eventedPlegRelistPeriod,
 			RelistThreshold: eventedPlegRelistThreshold,
 		}
-		klet.pleg = pleg.NewGenericPLEG(klet.containerRuntime, eventChannel, genericRelistDuration, klet.podCache, clock.RealClock{})
+		klet.pleg = pleg.NewGenericPLEG(ctx, klet.containerRuntime, eventChannel, genericRelistDuration, klet.podCache, clock.RealClock{})
```

Comment:
Let's not pass a `context.Context`. That gives the wrong impression that cancellation might be supported.

Instead, pass a `klog.Logger`.